common/lib/xmodule/xmodule/html_module.py

```diff
@@ -98,7 +98,7 @@ class HtmlModuleMixin(HtmlBlock, XModule):
         'js': [
             resource_string(__name__, 'js/src/collapsible.js'),
             resource_string(__name__, 'js/src/html/imageModal.js'),
-            resource_string(__name__, 'js/common_static/js/vendor/draggabilly.pkgd.js'),
+            resource_string(__name__, 'js/common_static/js/vendor/draggabilly.js'),
```

Comment:
Good question. I changed the name of the file, then did a find/replace to pick up any references. Other than updating the filename, I assume things with this module would work as they did. This particular module is unrelated to this scope of work. Thoughts?